File: app/src/main/java/com/example/evgeny/setlist_mobile/model/Song.java
package com.example.evgeny.setlist_mobile.model;

/**
 * Created by Evgeny on 23.07.2018.
 */

public class Song extends BaseModel {
    public String name;
    public String info;
    public boolean tape;
    public Artist cover;

    public Song() {
        cover = new Artist();
    }
}

File: app/src/main/java/com/example/evgeny/setlist_mobile/data/entity/Song.kt
package com.example.evgeny.setlist_mobile.data.entity

import com.example.evgeny.setlist_mobile.data.Artist
import com.example.evgeny.setlist_mobile.setlists.BaseModel
import com.example.evgeny.setlist_mobile.setlists.SongListItem

data class Song(val name: String, val info: String?, val tape: Boolean, val cover: Artist?, val with: Artist?, var number: Int) : BaseModel(),
    SongListItem {

    override fun toString(): String {
        return "Song(name='$name', info='$info', tape=$tape, cover=$cover, with=$with, number=$number)"
    }
}